.travis.yml

```diff
@@ -6,13 +6,16 @@ rvm:
   - 2.4.5
   - 2.5.3
   - 2.6.4
+  - 2.7.2
 
 gemfile:
   - gemfiles/activerecord_4_2.gemfile
```

Comment:
I don't think rails 4.2 worth supporting. As well as 5.0 or 5.1. Everyone have migrated a long time ago. Others don't care much about new gem versions in general.